src/Compilers/Test/Core/Compilation/OperationTreeVerifier.cs

```diff
@@ -97,7 +97,16 @@ private void LogCommonProperties(IOperation operation)
 
             // Type
             LogString(", ");
-            LogType(operation.Type);
+            if (operation.Kind == OperationKind.None && operation.Language == LanguageNames.CSharp && operation is WorkaroundNoneOperation)
+            {
+                //TODO: this should be removed!
+                //We need to update the CSharp unit-tests to allow Types on NoneOperations
+                LogType(null);
```

Comment:
Just do this unconditionally if it's a `NoneOperation`.

Reply:
Some NoneOperations have a `Type` today - if we would do this unconditionally those tests would fail (and I don't want to adjust those "valid" tests).